metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -505,24 +525,29 @@ def _parse_bigquery_log_entries(
     def _parse_exported_bigquery_audit_metadata(
         self, audit_metadata_rows: Iterable[BigQueryAuditMetadata]
     ) -> Iterable[QueryEvent]:
+        num_total_audit_entries: int = 0
```

Comment:
why not directly update `self.report.num_total_audit_entries` instead of using these local variables?

Reply:
will fix